
class AbortException(BaseException):
    def __init__(self):
        pass